README.md

```diff
@@ -1,3 +1,143 @@
+| `attributes.df`        | `Boolean` | Drop frame indicator         |
+| `attributes.frameRate` | `Number`  | Frame rate in FPS            |
+
+### Methods
+
+Each SMPTE object provides the following methods:
+
+#### `addFromSeconds(seconds)` - [**chainable**](#chainable)
+Adds a number of seconds to the current timecode object.
+* `seconds`: `Number` indicating the seconds to be added.
+
+#### `subtractFromSeconds(seconds)` - [**chainable**](#chainable)
+Substracts a timecode or a frame count from the current SMPTE object.
+* `seconds`: `Number` indicating the seconds to be subtracted.
+
+#### `add(time, operation = 1)` - [**chainable**](#chainable)
+Adds a timecode or a frame count to the current SMPTE object.
+* `time`: `Number|String|SMPTE` indicating the value to be added.
+* `operation`: `Number` used to get the sign of the `time`.
+
+#### `subtract(time)` - [**chainable**](#chainable)
+Substracts a timecode or a frame count from the current SMPTE object.
+* `time`: `Number|String|SMPTE` indicating the value to be subtracted.
+
+#### `toString(format = undefined)`
+Returns a string with the SMPTE timecode representation.
+* `format`: `String` mask to form the string.
+
+#### `toDate()`
+Converts a SMPTE object to a date object.
+
+#### `isTimecodeFormatValid(timecode, df)` - **static**
+Checks if a timecode string is in a valid format.
+* `timecode`: `String` to be evaluated.
+* `df`: `Boolean` indicating if is drop frame representation.
+
+#### `isValidTimecode(timecode, fr = defaults.frameRate, df = defaults.dropFrame)` - **static**
+Checks if a timecode is valid according to SMPTE standard.
+* `timecode`: `String` to be evaluated.
+* `fr`: `Number` indicating the frame rate.
+* `df`: `Boolean` indicating if is drop frame representation.
+
+#### `frameCountFromTimecode(timecode, fr = defaults.frameRate, df = defaults.dropFrame)` - **static**
+Gets the frame count given a timecode string.
+* `timecode`: `String` timecode.
+* `fr`: `Number` indicating the frame rate.
+* `df`: `Boolean` indicating if is drop frame representation.
+
+#### `isFramerateSupported(framerate)` - **static**
+Checks if a frame rate is supported.
+* `framerate`: `Number` indicating the frame rate.
+
+### Default values
+You can easily change the default value accessing the static `default` object from the SMPTE class and change the `frameRate` or the `dropFrame` property.
+
+```javascript
+import SMPTE from 'smpte.js';
+
+SMPTE.defaults.frameRate = 29.97;
+SMPTE.defaults.dropFrame = true;
+```
+
+## Definitions
+
+### [Chainable](https://en.wikipedia.org/wiki/Method_chaining)
+Returns an object, allowing the calls to be chained together in a single statement without requiring variables to store the intermediate results.
+
+## Contributing
+All contribution is welcome, please feel free to open tickets and pull requests.
+
+## License
+[MIT.](https://github.com/fireworkweb/smpte.js/blob/master/LICENSE)
```

Comment:
You can just link `LICENSE` file: https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file